addon/components/listen-button.js

```diff
@@ -102,6 +102,9 @@ export default Component.extend({
 
   didUpdateAttrs: diffAttrs('isLive', function(changedAttrs, ...args) {
     this._super(...args);
+    if (typeof FastBoot !== 'undefined' || !this.element || !get(this, 'isExpandable')) {
```

Comment:
FYI I don't totally get this, is `FastBoot` only defined if it's being computed on the server?

Reply:
Yes, exactly. There's an example of this here: https://www.ember-fastboot.com/docs/addon-author-guide#browser-only-initializers